.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: c
+
+sudo: false
+
+env:
+    global:
+        - LOCALUA_NO_LUAROCKS=y
+        - LOCALUA_NO_COMPAT=y
+
+before_install:
+    - |
+        mkdir -p $HOME/localua
+        pushd $HOME/localua
+            curl "https://raw.githubusercontent.com/catwell/localua/master/localua.sh" -o localua.sh
```

Comment:
localua is currently maintained at https://github.com/oploadk/localua.

This line should be updated to

    curl "https://raw.githubusercontent.com/oploadk/localua/master/localua.sh" -o localua.sh

to avoid a 404.

Reply:
Thanks for the update URL.